Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        // Used with float.TryParse to make tests pass ("." instead of "," in tests).
+        static CultureInfo invC = CultureInfo.InvariantCulture;
```

Comment:
Why static?

Reply:
I'm not entirly sure how that object works but it was to make the object visible in the function. 
Now I moved it down and put it in the function and it works without static.

My understanding was that you read it once as a global (static) variable/setting and then use it thruout the program?